### ISSUE TITLE
new-5.22.2

### ISSUE THREAD

**Issue by rurban** [perl11/cperl#40]
Need to get the branch `new-5.22.2` passing, before I can test the other fixes: win32 #39 and dbg-globs #38 

```
    porting/corelist.t
    porting/manifest.t
    porting/pod_rules.t
    porting/podcheck.t
    porting/regen.t
    porting/utils.t
```

**Comment by rurban** [comment]
only `porting/pod_rules.t` is now failing, need to add support for cdelta.pod's
Done in branch `bugfix/gh40-cdelta`

**Comment by rurban** [comment]
Fixed with cb781190441f3f255cfb and 4f81fb06f470dd955d6dbd7e7562f17e2e4d854d
